### PR TITLE
Improve workflows page

### LINKED ISSUE #4303: Signal *not* to make all workflows inactive when a project is finished.
A few projects have done that. The result is a broken classify page. Some text in the lab should warn builders not make all workflow inactive.
In addition, it would be useful to automatically move a project to "paused" or "finished" when all data is retired.

### PR DESCRIPTION
**Pull request by simoneduca**
Staging branch URL: https://fix-4303.pfe-preview.zooniverse.org/

Fixes #4303, counts toward #4221. 

This is the first step to improve the workflow page. Moving the workflow settings from their current place (the visibility page) to the workflows seems a good idea, but I'll do that in a separate PR.
Changes:

1. Display workflow completeness.
2. Add note about making workflows inactive.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

File: app/pages/lab/workflows.jsx [hunk 1]
```diff
@@ -9,10 +9,11 @@ import Paginator from '../../talk/lib/paginator';
 
 const WorkflowsPage = (props) => {
   const renderWorkflow = ((workflow) => {
+    let progressPercentage = workflow.completeness * 100;
     return (
       <li key={workflow.id}>
         <Link key={workflow.id} to={props.labPath(`/workflows/${workflow.id}`)} className="nav-list-item" activeClassName="active">
-          {workflow.display_name}
+          {workflow.display_name}{' -- '}{`${progressPercentage.toFixed(0)} % Complete`}
```
Comment (srallen) [inline app/pages/lab/workflows.jsx]: I'm not too keen on this. It makes it look like the percentage is a part of the display name.

File: app/pages/lab/workflows.jsx [hunk 2]
```diff
@@ -33,8 +34,9 @@ const WorkflowsPage = (props) => {
       <p>A workflow is the sequence of tasks that you’re asking volunteers to perform.</p>
       <p>An asterisk (*) denotes a default workflow.</p>
       <p>If you have multiple workflows you can rearrange the order in which they are listed on your project's front page by clicking the reorder view button and then clicking and dragging on the left gray tab next to each workflow title listed below.</p>
+      <p>NOTE: Please leave at least one active workflow; even if all workflows are 100% complete.</p>
```
Comment (srallen) [inline app/pages/lab/workflows.jsx]: Perhaps bold lettering can be used for emphasis instead of capitals?

Comment (simoneduca) [reply app/pages/lab/workflows.jsx]: Sure.

File: app/pages/lab/workflows.jsx (hunk 1 above)
Comment (simoneduca) [reply app/pages/lab/workflows.jsx]: Fair enough. Turning the list into a table should take care of that.

**Comment by simoneduca** [comment]
Looking into why that broke tests.

**Comment by simoneduca** [comment]
The test fails because it's looking for class name `.nav-list li`, which isn't suitable anymore for a table.

**Comment by eatyourgreens** [comment]
Looks like re-ordering workflows doesn't work any more, either.

**Comment by simoneduca** [comment]
`table` really doesn't play well with `DragReorderable` 😭

**Comment by eatyourgreens** [comment]
@simoneduca works for me with `<DragReorderable tag="table"`. What errors are you getting?

**Comment by simoneduca** [comment]
@eatyourgreens works here too, but I get this warning `<tr> cannot appear as a child of <table>. See DragReorderable > table > tr. Add a <tbody> to your code to match the DOM tree generated by the browser.` Not sure if that might lead to something unpredictable, but it seems not.

**Comment by CKrawczyk** [comment]
If it helps, I have used `DragReorderable` for a table for the external links editor page before: https://github.com/zooniverse/Panoptes-Front-End/blob/master/app/pages/lab/external-links-editor.jsx

**Comment by simoneduca** [comment]
Aha! thanks @CKrawczyk that's helpful.

**Comment by simoneduca** [comment]
Perhaps not the neatest way but the warning is gone now.

**Comment by srallen** [comment]
The reordering view doesn't need to be a table. I don't think project owners would need the tabular data and settings for the workflows when reordering, probably just the display names. But if you've already gotten it to work, I guess it doesn't need to change back.

**Comment by simoneduca** [comment]
Yeah it should be working now.

**Comment by simoneduca** [comment]
@srallen are you happy with this?

**Comment by simoneduca** [comment]
Good points! Thanks @srallen

**Comment by simoneduca** [comment]
Restaged. Also doesn't look it needs rebasing.

**Comment by simoneduca** [comment]
I borked a rebase. Hang in there.

**Comment by simoneduca** [comment]
Done, latest client version and test workflow-selection are in there now.

**Pull request merged**